approximate_string_matching/matching_with_mismatches.py

```diff
@@ -157,3 +157,68 @@ def _pad_power_2(w, m):
         if v + i > m:
           patterns[i][j] = m + 1
     yield from _text_matching(t, w, n, m, k, patterns)
+
+def bitap_shift_add(text: str, word: str, _n: int, m: int, k: int):
```

Comment:
Since we don't need the length of text in the algorithm, the parameter is unused, which causes issues with `pylint`. Underscore signifies the param is unused.

Reply:
Oh, right, I missed that, thank you.